src/fontra/views/editor/sidebar.js

```diff
@@ -4,6 +4,8 @@ import * as html from "/core/html-utils.js";
 export const MIN_SIDEBAR_WIDTH = 200;
 export const MAX_SIDEBAR_WIDTH = 500;
 
+const panelElementsPostionRight = ["selection-info"];
```

Comment:
The sidebars are very dynamic: it should not be needed to hard-code the "right"-ness of the selection-info panel. sidebar.identifier is "left" or "right", so you can deduce the alignment from that.

Reply:
Thanks for the hint. Makes sense. I changed it.